### PR TITLE
Added the `value` attribute for pselect

### PR DESCRIPTION
**Pull request by IonelLupu**
## What was done
The `<pselect>` can now accept the `value` attribute

The `value` property was added in order to programmatically set a value for the select input. This property is also available for all the other input elements like `pinput-text`,`pinput-numeric`,`ptextarea`, `ptoggle` etc. 

It can be used to set a default value, but most importantly, it allows you to get the value out of the `pselect`

### CODE REVIEW

File: src/resources/elements/primeDesignSystem/pselect/pselect.ts [hunk 1]
```diff
@@ -34,36 +34,33 @@ export interface IPSelectItemConfig {
...
-    if (this.data) {
-      selectConfig.data = [
+      data: [
         {text: this.placeholder, placeholder: true},
         ...this.data,
-      ];
-    }
-
-    this.select = new SlimSelect(selectConfig);
+      ],
```
Comment (hiaux0) [inline src/resources/elements/primeDesignSystem/pselect/pselect.ts:60]: How is this related to the `value` attribute addition?

File: src/resources/elements/primeDesignSystem/pselect/pselect.ts [hunk 2]
```diff
@@ -34,36 +34,33 @@ export interface IPSelectItemConfig {
...
+      onChange: info => this.value = info instanceof Array ? info.map(item => item.value) : info.value,
```
Comment (hiaux0) [inline src/resources/elements/primeDesignSystem/pselect/pselect.ts:61]:
```suggestion
      onChange: info => this.value = Array.isArray(info) ? info.map(item => item.value) : info.value,
```

Nit: Maybe just preference.

File: src/resources/elements/primeDesignSystem/demos/pselectDemo.html [hunk 3]
```diff
@@ -1,7 +1,10 @@
 <template>
   <section class="demo">
     <h1>Simple Dropdown</h1>
-    <pselect data.bind="simpleData"></pselect>
+    --${test}--
+    <pselect data.bind="simpleData" value.bind="test"></pselect>
```
Comment (hiaux0) [inline src/resources/elements/primeDesignSystem/demos/pselectDemo.html:5]: Obsolete dev code?

Could we have a demo, where an array of `value`s is set?

Comment (dkent600) [reply src/resources/elements/primeDesignSystem/demos/pselectDemo.html:5]: what is `  --${test}--` for?

**Review by dkent600 — CHANGES_REQUESTED** [review]
@IonelLupu Can you describe in the description why we care about `value`?  If this is for a default, maybe we can call it `default` or something like that?

File: src/resources/elements/primeDesignSystem/pselect/pselect.ts (hunk 2 above)
Comment (IonelLupu) [reply src/resources/elements/primeDesignSystem/pselect/pselect.ts:61]: didn't know about. love it

File: src/resources/elements/primeDesignSystem/pselect/pselect.ts (hunk 1 above)
Comment (IonelLupu) [reply src/resources/elements/primeDesignSystem/pselect/pselect.ts:60]: I couldn't type the `selectConfig` object in order to get intellisense so I decided to instantiate SlimSelect using the object directly. This way I prevented possible errors when adding SlimSelect properties.

**Comment by IonelLupu** [comment]
@dkent600 I updated the description of this PR
@hiaux0  I also added the `multiple` feature for the `pselect`

**Review by dkent600 — APPROVED** [review]
I see this isn't just about adding `value`, but also about multiple.  Should probably mention that in the description

**Comment by IonelLupu** [comment]
@dkent600 updated the description
